src/argus_htmx/management/commands/tailwind_config.py

```diff
@@ -12,38 +15,107 @@
 # Copied from https://github.com/GEANT/geant-argus/pull/15 with minor modifications
 class Command(BaseCommand):
     help = """\
-    
-Uses the 
-template specified in the TAILWIND_CONFIG_TEMPLATE setting (default: tailwind.config.js)
-to dynamically build a tailwind.config.js. The template should contain:
- - a'{{ daisyuithemes }}' section without square brackets that will be popuplated by the 
+Uses the template specified in the TAILWIND_CONFIG_TEMPLATE setting (default: tailwind/tailwind.config.js)
```

Comment:
Where's the root? Are we at repo root or somewhere in the code? What are we relative *to*?

Reply:
It's a template path, so it's whatever template is found by the template engine with that path

Reply:
Could you add that info?

Reply:
done